web/src/app/environments/view/environments-view.component.html

```diff
@@ -130,8 +130,11 @@ <h3 matLine>{{ environment.latestPing.url }}</h3>
           <span class="mat-caption text-right" flex="30" layout-align="row" *ngIf="!environment.latestPing.isValid">
             <p>No pings</p>
           </span>
+          <span class="mat-caption text-right" flex="30" layout-align="row" *ngIf="environment.latestPing.isValid">
```

Comment:
With both conditions (this one and the one above), we need to check if there are any pings available, in the variable environment.pings. I think this was me originally making a copy and paste error